Source/FullScreenPokemon.d.ts

```diff
@@ -1056,7 +1056,17 @@ declare module FullScreenPokemon {
         /**
          * Activates a Function to perform an HM move outside of battle.
          */
-        partyActivate?: (player: IPlayer, pokemon: IPokemon) => boolean;
+        partyActivate?: (player: IPlayer, pokemon: IPokemon) => void;
+
+        /**
+         * Activates a Function to make sure an HM move can be used outside of battle.
+         */
+        partyActivateCheckThing?: (player: IPlayer, pokemon: IPokemon, move: IMoveSchema) => void;
+
+        /**
+         * The HMSolid that the move affects.
+         */
+        solid?: IHMSolid;
```

Comment:
This should be a string title. IHMSolid refers to in-game objects, which doesn't make sense in this context.